pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/BrevbakerApp.kt

```diff
@@ -103,6 +121,18 @@ fun Application.brevbakerModule() {
         maxRetries = brevbakerConfig.propertyOrNull("pdfByggerMaxRetries")?.getString()?.toInt() ?: 30,
     )
 
+    FeatureToggleHandler.configure {
+        with(brevbakerConfig.config("unleash")) {
+            appName = stringProperty("appName")
+            environment = stringProperty("environment")
+            this@configure.host = stringProperty("host")
+            apiToken = stringProperty("apiToken")
+        }
+    }
+
     configureMetrics()
     brevbakerRouting(jwtConfigs.map { it.name }.toTypedArray(), latexCompilerService)
-}
\ No newline at end of file
+    log.warn("Ferdig med å sette opp applikasjonen")
```

Comment:
Skal dette være en warning?
Det er forøvrig sikkert mer riktig å logge det med
`monitor.subscribe(ServerReady) { it.log.info("Ferdig med å sette opp applikasjonen") }`